README.md

```diff
@@ -15,3 +15,36 @@ making this approach much faster to make changes to the PDF.
 ```sh
 oc port-forward -n crc-pdf-generator svc/crc-pdf-generator-api 8000:8000
 ```
+
+# Endpoints
+You currently have 2 choices for generating any of the available templates:
+
+The generate endpoint will automatically download the template if it is available (requires the above mentioned headers)
+* `POST http://localhost:8000/api/crc-pdf-generator/v1/generate` 
+
+The preview endpoint will instead return the pdf preview environment:
+* `GET http://localhost:8000/preview`
```

Comment:
The preview is not accessible in the deployed environment. We should mention that something.

Reply:
So just to clarify, that endpoint is exclusively for local development?

Reply:
Yes. 3scale will expose only routes under: /api/crc-pdf-generator/v1/

And yes the plan is to expose this preview endpoint as well sometime in the future. And that future is mainly coming after we are on 3scale gateway.